mne/viz/evoked.py

```diff
@@ -1778,20 +1778,20 @@ def _set_ylims_plot_compare_evokeds(ax, any_positive, any_negative, ymin, ymax,
 
     fraction = 2 if ax.get_ylim()[0] >= 0 else 3
 
-    if truncate_yaxis is not False:
+    if truncate_yaxis:
         _, ymax_bound = _truncate_yaxis(
             ax, ymin, ymax, orig_ymin, orig_ymax, fraction,
             any_positive, any_negative, truncate_yaxis)
     else:
-        if truncate_yaxis is True and ymin is not None and ymin > 0:
+        if ymin is not None and ymin > 0:
             warn("ymin is all-positive, not truncating yaxis")
         ymax_bound = ax.get_ylim()[-1]
 
     current_ymin = ax.get_ylim()[0]
 
     # plot v lines
     # Why 'invert_y'? Many EEG people plot negative values up for ... reasons
-    if invert_y is True and current_ymin < 0:
+    if invert_y and (current_ymin < 0):
```

Comment:
ibid, PEP8

Reply:
IIRC MNE conventions are different?

Reply:
AFAIK the only time we tend to put parens is when there is assignment and equals like
```
a = (b == c)
```
To prevent / make clear something accidental is not going on.

Reply:
OK, the parens are my own preference, you can remove them.  But def. remove the `is True`.

Reply:
No I meant `if x is True` instead of just `if x`.

Reply:
We used to have `if x is True` all over the place but nowadays we tend to do `if x` to allow things like 0/1 acting as True/False

Reply:
Oh great. That always bothered me.